### DISCUSSION TITLE
Does artillery playwright publish metrics to honeycomb or any other platform?

### DISCUSSION THREAD

**Discussion by sambordo1** [artilleryio/artillery#1913] [Artillery]
I use playwright for E2E tests and have recently integrated artillery with them to develop load tests. I am researching the best monitoring and reporting platforms to stack with playwright and artillery. I am considering honeycomb but was unsure if it works with artillery playwright. I see [here](https://www.artillery.io/docs/guides/plugins/plugin-publish-metrics) the supported monitoring targets for artillery. Do any of those work with artillery playwright or do they only work for HTTP tests at the protocol level ?

**Comment by hassy** [comment]
hi @sambordo1 👋 The Honeycomb integration only works with tests using the HTTP engine right now. Adding support for Playwright would be pretty straightforward though - what type of information would you like to have access to in Honeycomb traces?

**Reply by sambordo1** [reply]
hi @hassy after further discussion, I think I will try Cloudwatch for publishing metrics. Thank you for the response!
